src/figuresClasses.ts

```diff
@@ -1,11 +1,82 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: Function;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape = 'triangle',
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('Invalid parameters.');
+    }
+
+    const sides: number[] = [sideA, sideB, sideC].sort((a, b) => b - a);
+
+    if (sides[0] >= sides[1] + sides[2]) {
+      throw new Error(
+        `sides ${sideA}, ${sideB} and ${sideC} can't form a triangle`,
+      );
+    }
+  }
+
+  getArea(): number {
+    const semiper: number = (this.sideA + this.sideB + this.sideC) / 2;
+
+    return +(
+      (semiper *
+        (semiper - this.sideA) *
+        (semiper - this.sideB) *
+        (semiper - this.sideC)) **
+      0.5
+    ).toFixed(2);
+  }
+}
 
-export class Triangle implements Figure {}
+export class Circle implements Figure {
+  constructor(
+    public color: string,
+    public radius: number,
+    public shape = 'circle',
+  ) {
+    if (radius <= 0) {
+      throw new Error('Invalid parameters.');
+    }
+  }
 
-export class Circle implements Figure {}
+  getArea(): number {
+    const result: number = Math.PI * this.radius ** 2;
+
+    // return +result.toFixed(2);
+
+    return Math.floor(result * 100) / 100;
+  }
+}
+
+export class Rectangle implements Figure {
+  constructor(
+    public color: string,
+    public width: number,
+    public height: number,
+    public shape = 'rectangle',
+  ) {
+    if (width <= 0 || height <= 0) {
```

Comment:
[CODE STYLE] Similarly, the error message for the Rectangle class should be more descriptive, such as 'Rectangle width and height must be greater than zero.'